packages/apps/plugins/plugin-inbox/src/components/Email/MessageList.tsx

```diff
@@ -96,13 +99,33 @@ export const MessageItem = ({ message, selected, onSelect, onAction }: MessageIt
                 </Button>
               </div>
             )}
-            <div className={mx('shrink-0 whitespace-nowrap p-2', onAction && 'group-hover:hidden')}>
+            <div className={mx('shrink-0 whitespace-nowrap p-2 text-sm', onAction && 'group-hover:hidden')}>
               {formatDate(new Date(), date)}
             </div>
           </div>
-          <div className={mx('mb-1 mr-2 overflow-hidden line-clamp-3', message.read && 'text-neutral-500')}>
+
+          <div
+            className={mx('mb-1 mr-2 overflow-hidden line-clamp-3 cursor-pointer', message.read && 'text-neutral-500')}
+            onClick={() => setExpanded((e) => !e)}
+          >
             {subject}
           </div>
+
+          {expanded && (
+            <div className='flex flex-col gap-2 pbs-2 mt-2 border-bs-2 separator-separator border-description'>
+              {message.blocks.map((block, index) => (
+                <React.Fragment key={index}>
+                  <div className='grid grid-cols-[1fr,9rem] gap-2 fg-description'>
+                    <div className='text-sm'>{block.content?.content}</div>
+                    <div className='text-xs text-right'>{formatDate(new Date(), new Date(block.timestamp))}</div>
```

Comment:
Nit: Technically create const now = Date.now() at the start of the function and use that.